combinetf2/fitter.py

```diff
@@ -654,9 +748,16 @@ def _compute_yields_noBBB(self, compute_norm=False, full=True):
             logsnorm = tf.matmul(mlogk, mthetaalpha)
             logsnorm = tf.reshape(logsnorm, [nbins, self.indata.nproc])
 
-            snorm = tf.exp(logsnorm)
+            if self.indata.systematic_type == "log_normal":
+                snorm = tf.exp(logsnorm)
+                snormnorm = snorm * norm
+            elif self.indata.systematic_type == "normal":
+                snormnorm = norm + logsnorm
+            else:
+                raise RuntimeError(
```

Comment:
Do we need to check this downstream everytime? Suggest to check upstream in indata once

Reply:
Done